src/test/java/com/microsoft/azure/proton/transport/proxy/impl/ProxyHandlerImplTest.java

```diff
@@ -52,6 +52,121 @@ public void testValidateProxyResponseOnSuccess() {
 
         Assert.assertEquals(0, buffer.remaining());
     }
+	
+	    @Test

```

Comment:
This test duplication can be reduced using Parameterized tests. https://github.com/junit-team/junit4/wiki/Parameterized-tests
nit: `@Test` should align with `public` keyword.

Reply:
Okay added a new library junit-jupiter-params and created a parameterized test class to test all http codes defined on the RFC 	@ValueSource(ints = {200, 201, 202, 203, 204, 205, 206})

hope with this the pull request is approved and this can be released, need to stress that thsi is really important for us